src/constraints/outage_constraints.jl

```diff
@@ -276,77 +286,95 @@ end
 function add_MG_storage_dispatch_constraints(m,p)
```

Comment:
I renamed this function to specify elec storage. The constraints should also be updated to loop b in p.s.storage.types.elec instead hardcoding "ElectricStorage". Like H2 MG state of charge constraint below.